packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -74,6 +76,7 @@ const IdvFailed = ({
     account_settings,
     handleSubmit,
     mismatch_status = idv_error_statuses.poi_failed,
+    chosen_country,
```

Comment:
```suggestion
...rest
```

Reply:
Just for one prop do we need to use rest ?

Reply:
I'll swap the names inside the component as the variable is local to the component